test/e2etest/test-config.tfvars.tmpl

```diff
@@ -143,3 +143,20 @@ eks_additional_roles = [
   }
 ]
 {{end}}
+
+################################################################################
+# Crowd Settings
+################################################################################
+# remove this once Crowd Helm chart is fixed
+crowd_version_tag = "5.1.2"
```

Comment:
Get rid of this one when https://github.com/atlassian/data-center-helm-charts/pull/532 is merged and released.